### DISCUSSION TITLE
JS alert popup close in Seleniumbase without any actions.

### DISCUSSION THREAD

**Discussion by SergeyGuschin** [seleniumbase/SeleniumBase#2428] [General]
Hello. In Seleniumbase when I click on the delete button it should be JS alert popup message appears and waits for it to be accepted. But when I create a tour in Seleniumbase and click the delete button the JS alert window opens and closes immediately without any actions from my side and I can't intercept and accept this alert with the `self.wait_for_and_accept_alert()` command. How can achieve that when  I click on the delete button and the JS alert popup does not close immediately? Thank you

**Comment by mdmintz** [comment]
Instead of `self.click(selector)`, use `self.find_element(selector).click()` so that the alert stays open after the click.

**Reply by SergeyGuschin** [reply]
Thank you so much. You are the best as always.
